components/ProfilePage/FollowingTab.tsx

```diff
@@ -44,23 +46,49 @@ export const Styled = styled.div`
 export function FollowingTab({ className }: Props) {
   const { user } = useAuth()
   const uid = user?.uid
-  const subscriptionRef = collection(firestore, `/users/${uid}/subscriptions/`)
+  const subscriptionRef = collection(
+    firestore,
+    `/users/${uid}/activeTopicSubscriptions/`
+  )
+
   let billList: string[] = []
-  const [unfollowModal, setUnfollowModal] = useState<"show" | null>(null)
-  const [currentBill, setCurrentBill] = useState<string>("")
   const [billsFollowing, setBillsFollowing] = useState<string[]>([])
+  let orgsList: string[] = []
+  const [orgsFollowing, setOrgsFollowing] = useState<string[]>([])
+
+  const [currentCourt, setCurrentCourt] = useState<number>(0)
+  const [currentOrgName, setCurrentOrgName] = useState<string>("")
+  const [currentType, setCurrentType] = useState<string>("")
+  const [currentTypeId, setCurrentTypeId] = useState<string>("")
+  const [unfollowModal, setUnfollowModal] = useState<"show" | null>(null)
```

Comment:
These pieces of state are used to configure the modal, correct? since they're always set and accessed together, consider consolidating into a single `useState<UnfollowModalConfig | null>` piece. Show the modal if the config is present.

```
type UnfollowModalConfig = {
  court: number,
  orgName: string,
  currentType: string,
  currentTypeId: string
}
...

const [unfollow, setUnfollow] = useState<UnfollowModalConfig | null>(null)
```

Reply:
done